hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/BaseTestKafkaSource.java

```diff
@@ -261,7 +261,7 @@ public void testFailOnDataLoss() throws Exception {
     assertEquals(2, fetch1.getBatch().get().count());
 
     // wait for the checkpoint to expire
-    Thread.sleep(10001);
+    Thread.sleep(30000);
```

Comment:
Given that the sleep time is reduced inside the `KafkaOffsetGen`, I need to adjust the wait time in this functional test case to make sure it does not fail.  Note that this does not increase the overall test running time (originally the test takes 43s; now it takes 33s, even increasing the sleep time here).